### ISSUE TITLE
Refinements

### ISSUE THREAD

**Issue by alexanderdittrich** [intelligent-soft-robots/pam_interface#7]
I looked through the repo a little bit more closely. Some things i noticed:

- `command.cpp` in include directory
- two license files (LICENSE, license.txt) with same content (BSD 3-Clause License)
- in the bin directory: is it intended that the files do not have a file-format (e.g. *.py)? Files include python code for visualization. 
- more a question: `pybind11` wraps C++11 code and python, correctly? However, the cmake-file compiles with C++17 (`CXX_STANDARD 17`). Can this cause potential issues?
- `add_documentation() `in cmake

**Comment by vincentberenz** [comment]
@alexanderdittrich 

- that seems to be a mistake, most likely this file can be deleted
- only one file should be kept, not sure which one
- yes, because it could be argued that the person executing the file does not need to know the underlying code is in python. This being said, there is an "install_script" cmake macro that would be better. It allows the sources file to have a python extension, but the executable once installed not having the extension anymore. For example: https://github.com/intelligent-soft-robots/o80_pam/blob/master/CMakeLists.txt#L90
- pybind11 works also for c++17
- indeed, should be added !

**Comment by alexanderdittrich** [comment]
@vincentberenz 
What is the use of `void copy(const RobotState<NB_DOFS>& other);` in `state/robot.hpp`. I could not find a fitting declaration of copy.

**Comment by vincentberenz** [comment]
> @vincentberenz What is the use of `void copy(const RobotState<NB_DOFS>& other);` in `state/robot.hpp`. I could not find a fitting declaration of copy.

Most likely it could be removed :)

**Comment by vincentberenz** [comment]
> > @vincentberenz What is the use of `void copy(const RobotState<NB_DOFS>& other);` in `state/robot.hpp`. I could not find a fitting declaration of copy.
> 
> Most likely it could be removed :)

yes, let's try and see how to goes :)

**Comment by vincentberenz** [comment]
> @vincentberenz Just documented the general configuration class (`include/pam_interface/configuration.hpp`). Just for interest: Is it intended that the smallest maximum pressure parameter is returned by the function `max_pressure()`

@alexanderdittrich 
you mean here ?
https://github.com/intelligent-soft-robots/pam_interface/blob/master/include/pam_interface/configuration.hxx#L81
looks like a bug !

**Comment by alexanderdittrich** [comment]
> 

Yes. I was not sure, if the function is only ambiguous named (but intended implementation) or buggy (potentially copy paste error for `min_pressure()`).

I would suggest that the bug should be tested on the robot before changing, if it is not clear of smallest maximum pressure parameter is intended.
